src/Label.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import cx from 'classnames';
+
+/**
+ * @module Label
+ */
+class Label extends React.Component {
+	render() {
+		const {
+			inputId,
+			text,
+			isRequired,
+			children,
+			className,
+			...other
+		} = this.props;
+
+		const classNames = cx(
+			{ required : isRequired },
+			className
+		);
+
+		return (
+			<label className={classNames}
+				htmlFor={inputId}
+				{...other}>
+				{text}
+				{children}
+			</label>
+		);
+	}
+}
+
+Label.propTypes = {
+};
+
+export default Label;
```

Comment:
ok. I'd rather not create another component if its not necessary. @adamd do you agree with Mike? I can just keep `label` and html element then. 

@mikem-mup any thought on breaking out the html text input into its own component while TextField (can rename) would be something more compound with label, error, and possibly char counter?